configs/desktop_savant_events_schemas.json

```diff
@@ -0,0 +1,1122 @@
+          }
+        },
+        {
+          "name": "remove bookmark",
+          "description": "client removes a bookmark",
+          "amplitudeProperties": {
+            "meta-cat-1": "'feature'",
+            "meta-cat-2": "'bookmark'"
+          },
+          "schema": {
+            "$schema": "http://json-schema.org/schema#",
+            "type": "object",
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0
+              },
+              "category": {
+                "type": "string",
+                "enum": [
+                  "savant"
+                ]
+              },
+              "method": {
+                "type": "string",
+                "enum": [
+                  "bookmark"
+                ]
+              },
+              "object": {
+                "type": "string",
+                "enum": [
+                  "remove"
+                ]
+              }
+            },
+            "required": [
+              "timestamp",
+              "category",
+              "method",
+              "object"
+            ]
+          }
+        },
+        {
+          "name": "open hamburger menu",
+          "description": "client opens the hamburger menu",
+          "amplitudeProperties": {
+            "meta-cat-1": "'feature'"
+          },
+          "schema": {
+            "$schema": "http://json-schema.org/schema#",
+            "type": "object",
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0
+              },
+              "category": {
+                "type": "string",
+                "enum": [
+                  "savant"
+                ]
+              },
+              "method": {
+                "type": "string",
+                "enum": [
+                  "hamburger_menu"
+                ]
+              }
+            },
+            "required": [
+              "timestamp",
+              "category",
+              "method"
+            ]
+          }
+        },
+        {
+          "name": "open library menu",
+          "description": "client opens the library menu",
+          "amplitudeProperties": {
+            "meta-cat-1": "'feature'"
+          },
+          "schema": {
+            "$schema": "http://json-schema.org/schema#",
+            "type": "object",
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0
+              },
+              "category": {
+                "type": "string",
+                "enum": [
+                  "savant"
+                ]
+              },
+              "method": {
+                "type": "string",
+                "enum": [
+                  "library_menu"
+                ]
+              }
+            },
+            "required": [
+              "timestamp",
+              "category",
+              "method"
```

Comment:
Same kind of thing here, re: the object (I do think it's worth explicitly adding since if events.yaml changes and we add a "close" event then we'd catch both of those)